nginx/config/nginx.conf

```diff
@@ -30,60 +26,44 @@ http {
     include             /etc/nginx/mime.types;
     default_type        application/octet-stream;
 
-    # Load modular configuration files from the /etc/nginx/conf.d directory.
-    # See http://nginx.org/en/docs/ngx_core_module.html#include
-    # for more information.
-    include /etc/nginx/conf.d/*.conf;
-
     server {
         listen       80 default_server;
         listen       [::]:80 default_server;
         server_name  multinet.app;
- 
-        # Load configuration files for the default server block.
-        include /etc/nginx/default.d/*.conf;
-
-        location / {
-           proxy_pass http://127.0.0.1:8080/;
-        }
-
-        location /nodelink/ {
-           proxy_pass http://127.0.0.1:8081/;
-        }
-
-        location /adjmatrix/ {
-           proxy_pass http://127.0.0.1:8082/;
-        }
-
-        location /webhook/ {
-           proxy_pass http://127.0.0.1:4112/;
-        }
-
-        error_page 404 /404.html;
-            location = /40x.html {
-        }
-
-        error_page 500 502 503 504 /50x.html;
-            location = /50x.html {
-        }
```

Comment:
How do we serve these other apps if these declarations are being removed?

Reply:
They're still defined in the https block below and this http block now redirects to the https block

Reply:
Ah I see, thanks!